README.md

```diff
@@ -201,5 +201,90 @@ to check that it hasn't crashed.
 
 Congratulations, you have just run your first GRChombo simulation!
 
+### Visualisation
+
+In this section, we will outline how to get set up with visualisation using
+Paraview on Fawcett. It is a bit fiddly, so we will go over the bare minimum
+to get up and running. For more detailed information on what Paraview can do,
+have a look at the
+[tutorial](https://www.paraview.org/Wiki/images/b/bc/ParaViewTutorial56.pdf)
+(or ask Amelia!). Adapted from instructions provided by Kacper Kornet.
+
+First, we need to set up ssh access to fawcett. Information on how to do this
+is given [here](https://www.maths.cam.ac.uk/computing/fawcett-ssh-access). We
+have sent this to you in advance, so you should have all of the steps set up
+prior to the tutorial.
+
+Second, we need you to download Paraview v5.6.0 onto your local machine i.e. the
+laptop/desktop you are using to access fawcett. This can be done
+[here](https://www.paraview.org/download/). Again, we will have asked you to
+do this in advance.
+
+Before we tunnel to fawcett, we need to add a server on the Paraview
+client. To do this, open Paraview, and click 'connect' in the top left corner,
+then `add server'. Change the name to whatever you like e.g. 'localhost'. All
+other settings should be the default
+
+Server type: Client/Server
+Host: localhost
+Port: 11111
+
+Click Configure. On the next screen, set 
+
+Startup Type: Manual
+
+Accept by clicking Save and Close. Once you have set this up, you won't need
+to do it again.
+
+Now we need to start the Paraview server on fawcett. Ssh into fawcett - as
+outlined above, this should already be configured with ssh forwarding, and so
+shouldn't require explicit login via another machine. Load Paraview using
+
+```bash
+module load paraview/5.6.0/upstream
+```
+
+Now run the command
+```bash
+pvserver
+```
+
+It should print something like:
+
+```bash
+Waiting for client...
+Connection URL: cs://fawcett:11111
+Accepting connection(s): fawcett:11111
+```
```

Comment:
The `hostname` on fawcett is `mn01` and not `fawcett` (disappointingly).
````suggestion
```bash
Waiting for client...
Connection URL: cs://mn01:11111
Accepting connection(s): mn01:11111
```
````